client/package.json

```diff
@@ -24,16 +25,14 @@
     "vue-fragment": "^1.5.1",
     "vue-router": "^3.0.7",
     "vue-shortkey": "^3.1.4",
-    "vue-test-utils": "^1.0.0-beta.11",
```

Comment:
This would be a dev-only change, but I couldn't find anywhere this was used so I deleted it.